backend/src/main/java/com/itlize/joolemarketplace/controller/DescriptionController.java

```diff
@@ -0,0 +1,87 @@
+package com.itlize.joolemarketplace.controller;
+
+import com.itlize.joolemarketplace.model.Description;
+import com.itlize.joolemarketplace.model.Product;
+import com.itlize.joolemarketplace.service.DescriptionService;
+import com.itlize.joolemarketplace.service.ProductService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/descriptions")
+public class DescriptionController {
+    @Autowired
+    DescriptionService descriptionService;
+
+    @Autowired
+    ProductService productService;
+
+    @PostMapping
+    public ResponseEntity<?> createDescription(@RequestBody Description description) {
+        try {
+            Description createdDescription = descriptionService.createDescription(description);
+            return new ResponseEntity<>(createdDescription, HttpStatus.CREATED);
+        }
+        catch (RuntimeException e) {
+            Map<String, String> responseBody = new HashMap<>();
+            responseBody.put("error", "Description already exists");
+            responseBody.put("details", e.getMessage());
+            return new ResponseEntity<>(responseBody, HttpStatus.CONFLICT);
+        }
+    }
```

Comment:
I'm curious how this endpoint works without the `productId`. Were you able to test the endpoint?